src/verbosity.jl

```diff
@@ -18,51 +18,56 @@ vlogmacrodocs = """
 
 "Verbose" logging macros. Drop in replacements of standard logging macros, but an
 additional verbosity level `N` is passed to indicate differing verbosity levels
-for a given log level. The verbosity argument is subtracted from the base log level when passed down
-to the core logging logic, so `@debugv 1 msg` will essentially call `@logmsg Debug-1 msg`.
+for a given log level. The verbosity argument is passed as the `group` argument
+to the core logging logic as a `LoggingExtras.Verbosity` object.
 
-An `LoggingExtras.LevelOverrideLogger`can then be used to filter on the `level` argument.
+Note these "verbose" logging messages will only be filtered if a filter logger is used.
+A `LoggingExtras.EarlyFilteredLogger`can be used to filter on the `group.verbosity` argument.
 For convenience, the
-[`LoggintExtras.with(f; level, verbosity)`](@ref) function is provided to temporarily
-wrap the current logger with a log level and verbosity subtracted to filter while `f` is executed.
+[`LoggintExtras.withlevel(f, level; verbosity)`](@ref) function is provided to temporarily
+wrap the current logger with a log level and verbosity to filter while `f` is executed.
 """
 
+struct Verbosity
+    verbosity::Int
+end
+
 "$vlogmacrodocs"
 macro debugv(verbosity::Int, msg, exs...)
     return restore_callsite_source_position!(
-        esc(:($Base.@logmsg (Logging.Debug - $verbosity) $msg $(exs...))),
+        esc(:($Base.@debug $msg _group=LoggingExtras.Verbosity($verbosity) $(exs...))),
```

Comment:
Does the following work?

```suggestion
        :($Base.@debug $msg _group=$(Verbosity(verbosity)) $(esc.(exs)...))),
```

If this does work it would seem more hygienic, and less vulnerable to someone doing `using LoggingExtras as LogEx`
breaking things

Reply:
Yeah, that's a better idea.